modules/javafx.media/src/main/native/gstreamer/gstreamer-lite/gst-plugins-good/gst/spectrum/gstspectrum.c

```diff
@@ -877,6 +912,12 @@ gst_spectrum_transform_ip (GstBaseTransform * trans, GstBuffer * buffer)
   channels = GST_AUDIO_FILTER_CHANNELS (spectrum);
   bps = GST_AUDIO_FILTER_BPS (spectrum);
   bpf = GST_AUDIO_FILTER_BPF (spectrum);
+#ifdef OSX
```

Comment:
Should this also be qualified with `GSTREAMER_LITE` since it represents changes specific to our port?

Reply:
It doesn't look required to me, but it would make more sense and bring it at the same level as line 738

Reply:
No need for GSTREAMER_LITE, since this code already inside #ifdef GSTREAMER_LITE. See line 885 and 925.

Reply:
Yes, I see.